src/cache/cloud_fraction.jl

```diff
@@ -140,6 +155,56 @@ NVTX.@annotate function set_cloud_fraction!(
             ),
         )
     end
+
+end
+
+NVTX.@annotate function set_cloud_fraction!(
```

Comment:
Does it not work if you combine this with the function for diagnostic edmf?

Reply:
The issue is that the existing diagnostic EDMF `set_cloud_fraction!` function uses precomputed quantities (ie. `ᶜρaʲs` for updrafts) which aren't computed for prognostic edmf. To have a combined function I needed if/else statements for basically every operation in the function to handle these, so ended up just dispatching with 2 functions.

Reply:
Ah, right, I forgot about that. Something like this might work if you want to give it a try: https://github.com/CliMA/ClimaAtmos.jl/blob/66ad251dc5c9e7550ce0a2bf943e9717d0b20ab8/src/prognostic_equations/edmfx_tke.jl#L26